pr_inspector/config.py

```
@@ -0,0 +1,52 @@
+        config_path: Path to the configuration YAML file
+        
+    Returns:
+        Dictionary containing the configuration values
+    """
+    config_file = Path(config_path)
+    
+    if not config_file.exists():
+        raise FileNotFoundError(
+            f"Configuration file not found: {config_path}. "
+            "Please create config.yaml with server settings."
+        )
```

Ruff (0.14.10)

Avoid specifying long messages outside the exception class

(TRY003)